docs: add kubernetes version downgrade warning and hello-universe pack version DOC-1165 DOC-1174 (#2786)
spectrocloud/librarium: docs/docs-content/integrations/hello-universe.md

---

[Hello Universe](https://github.com/spectrocloud/hello-universe) is a demo web application utilized to help users learn
more about [Palette](https://docs.spectrocloud.com/introduction) and its features.

You can deploy it using two preset configurations:

- A standalone front-end application. It provides a click counter that is saved locally and displays Spectro Cloud
  themed images.

> - A three-tier application with a front-end application, API server and PostgreSQL database into a Kubernetes cluster.

Reviewer: ```suggestion
- A three-tier application with a front-end tier, API server, and PostgreSQL database.
```
maybe? :)

  It provides a click counter that is saved in the deployed database and displays Spectro Cloud themed images. You can
  read more about this configuration on the Hello Universe
  [README](https://github.com/spectrocloud/hello-universe?tab=readme-ov-file#reverse-proxy-with-kubernetes).

:::info

> The three-tier application mode is only supported by version 1.1.2.

Reviewer: ```suggestion
The three-tier application configuration is only supported by version 1.1.2 of the pack.
```

}
```

</TabItem>

<TabItem label="1.1.2" value="1.1.2">

## Prerequisites

- A Palette account.

- A cluster profile where the Hello Universe pack can be integrated.

- A Palette cluster with port `:8080` available. If port 8080 is not available, you can set a different port in the
  **values.yaml** file.

- If you are using the **Enable Hello Universe API** preset, you will need the `:3000` port available on your cluster
  too. Check out the [Usage](#usage) section for further details.

- Ensure sufficient CPU resources within the cluster to allocate a minimum of 500 milliCPU and a maximum of 500 milliCPU
  per replica.

## Parameters

The following parameters are applied to the **hello-universe.yaml** manifest through the **values.yaml** file. Users do
not need to take any additional actions regarding these parameters.

> | **Parameter**                       | **Description**                                                                                                                                  | **Default Value**                                                                              | **Required** |

Reviewer: the table looks broken :(

Author: Fixed! Thanks a lot ❤️

| ----------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------- | ------------ | --- | --- |
| `manifests.namespace`               | The namespace in which the application will be deployed.                                                                                         | `hello-universe`                                                                               | Yes          |

> | `manifests.images.hellouniverse`    | The [`hello-universe`](https://github.com/spectrocloud/hello-universe) application image that will be utilized to create the containers.         | `ghcr.io/spectrocloud/hello-universe:1.1.2`/ `ghcr.io/spectrocloud/hello-universe:1.1.2-proxy` | Yes          |

Reviewer: ```suggestion
| `manifests.images.hellouniverse`    | The [`hello-universe`](https://github.com/spectrocloud/hello-universe) application image that will be utilized to create the front-end containers.         | `ghcr.io/spectrocloud/hello-universe:1.1.2`/ `ghcr.io/spectrocloud/hello-universe:1.1.2-proxy` | Yes          |
```

> | `manifests.images.hellouniverseapi` | The [`hello-universe-api`](https://github.com/spectrocloud/hello-universe-api) application image that will be utilized to create the containers. | `ghcr.io/spectrocloud/hello-universe-api:1.0.12`                                               | No           |

Reviewer: ```suggestion
| `manifests.images.hellouniverseapi` | The [`hello-universe-api`](https://github.com/spectrocloud/hello-universe-api) application image that will be utilized to create the API containers. | `ghcr.io/spectrocloud/hello-universe-api:1.0.12`                                               | No           |
```

> | `manifests.images.hellouniversedb`  | The [`hello-universe-db`](https://github.com/spectrocloud/hello-universe-db) application image that will be utilized to create the containers.   | `ghcr.io/spectrocloud/hello-universe-db:1.0.2`                                                 | No           |

Reviewer: ```suggestion
| `manifests.images.hellouniversedb`  | The [`hello-universe-db`](https://github.com/spectrocloud/hello-universe-db) application image that will be utilized to create the database containers.   | `ghcr.io/spectrocloud/hello-universe-db:1.0.2`                                                 | No           |
```
